composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.3.2|^7.0",
-        "symfony/framework-bundle": "~2.0|^3.0",
+        "symfony/framework-bundle": "~2.0|^3.1",
```

Comment:
`^3.0` already supports 3.1 (and all releases <4.0). See https://getcomposer.org/doc/articles/versions.md#caret